upgrade/helpers/docker.py

```diff
@@ -30,12 +30,17 @@ def generate_satellite_docker_clients_on_rhevm(
             'Clients count to generate on Docker should be atleast 1 !')
         sys.exit(1)
     satellite_hostname = os.environ.get('RHEV_SAT_HOST')
-    ak = ak or os.environ.get('RHEV_CLIENT_AK_{}'.format(client_os.upper()))
+    ak = custom_ak or os.environ.get(
+        'RHEV_CLIENT_AK_{}'.format(client_os.upper()))
     result = {}
     for count in range(int(clients_count)):
         if bz_bug_is_open('1405085'):
             time.sleep(5)
-        hostname = '{0}dockerclient{1}'.format(count, client_os)
+        # If custom activation key is passed, it will be used to create custom
+        # docker clients for scenario tests and we will require to set distinct
+        # hostname for those content hosts
+        host_title = 'scenarioclient' if custom_ak else 'dockerclient'
+        hostname = '{0}{1}{2}'.format(ak, host_title, client_os)
```

Comment:
Please append count as well otherwise there will be duplication of clients with same for more that one count.

Reply:
will replace ak to use count